packages/core/src/components/Modal/Modal.tsx

```diff
@@ -170,40 +171,42 @@ const Modal: FC<ModalProps> & { width?: typeof ModalWidth } = ({
 
   const customWidth = width !== ModalWidth.DEFAULT && width !== ModalWidth.FULL_WIDTH;
 
-  const dialog = ReactDOM.createPortal(
-    <div
-      {...attr.container}
-      className={cx(styles.container, classNames.container)}
-      data-testid={dataTestId || getTestId(ComponentDefaultTestId.MODAL, id)}
-      style={{ "--monday-modal-z-index": zIndex }}
-    >
-      {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events */}
+  const dialog =
+    isClient() &&
+    ReactDOM.createPortal(
```

Comment:
seem like you can remove this `ReactDOM.createPortal` from here
it calls `createPortal` at [line 209](https://github.com/mondaycom/vibe/pull/2477/files#diff-912bdffbe0842863767b158e48535904a784f7bebe85bcd4dfcf42ef32b05f6fR209) as well

at the current implementation it would not render the Modal at all when SSR